.github/workflows/StaticTypeCheck.yml

```diff
@@ -35,19 +35,29 @@ on:
         required: false
         default: '-r tests/requirements.txt'
         type: string
-      report:
+      commands:
+        description: 'Commands to run the static type checks.'
+        required: true
+        type: string
+      html_report:
         description: 'Directory to upload as an artifact.'
         required: false
         default: 'htmlmypy'
         type: string
-      commands:
-        description: 'Commands to run the static type checks.'
-        required: true
+      junit_report:
+        description: 'junit file to upload as an artifact.'
+        required: false
+        default: 'StaticTypingSummary.xml'
         type: string
-      artifact:
+      html_artifact:
         description: 'Name of the typing artifact.'
```

Comment:
We should make this description different from the one of "junit_artifact".

Reply:
Fixed in 301584a670de754337a6f1c38e07f68cde79a656 and integrated into #51.